docs/api_doc.rb

```diff
@@ -0,0 +1,225 @@
+require 'factory_bot'
```

Comment:
Given the complexity of the documentation setup, (even in this new and exciting world of making any damn sense at all :-) I think it'd be good to have a file-level or class-level comment just generally saying something like "This is for pulling data attached to the ASpace API at class definition time and processing it via an ERB template into Slate API documentation (link to slate)"

Reply:
@pobocks Done - see latest revision. Thanks!